Cargo.toml

```diff
@@ -15,3 +15,4 @@ num-traits = "0.2"
 gc-arena = { path = "./gc-arena" }
 rustyline = "3.0.0"
 clap = "~2.32"
+num = "*"
```

Comment:
Why the addition of depending on num?  I don't think it's used anywhere?

Reply:
Unfortunately, it's used for its traits so that I didn't have to write an integer and a float version of safe_div.  Specifically, I'm using "Zero" and "ToPrimitive".

Reply:
oooh, I think I see what you're getting at, yeah, I should just pin to num-traits, and not pull in all of num.

Reply:
aaaand, num-traits is already in the dependencies.  Well, I feel somewhat foolish now.